apps/explorer/src/components/pagination/Pagination.tsx

```diff
@@ -111,12 +112,25 @@ function Pagination({
     );
 
     const Stats = stats ? (
-        <div>
-            {typeof stats.count === 'number'
-                ? numberSuffix(stats.count)
-                : stats.count}{' '}
-            {stats.stats_text}
-        </div>
+        !stats.loadState ? (
+            <div>
```

Comment:
qq why is there a loading state in the pagination component?

Reply:
While the Figma Design suggested having a pending state for the total count, the pagination is built using the total count, so I agree that, for now, we should remove this pending logic and just keep the Pagination component code as is.